src/vinyldns/client.py

```diff
@@ -622,12 +631,39 @@ def list_batch_change_summaries(self, start_from=None, max_items=None, **kwargs)
             args.append(u'startFrom={0}'.format(start_from))
         if max_items is not None:
             args.append(u'maxItems={0}'.format(max_items))
+        if ignore_access:
+            args.append(u'ignoreAccess={0}'.format(ignore_access))
+        if approval_status:
+            args.append(u'approvalStatus={0}'.format(approval_status))
 
         url = urljoin(self.index_url, u'/zones/batchrecordchanges') + u'?' + u'&'.join(args)
 
         response, data = self.__make_request(url, u'GET', self.headers, **kwargs)
         return ListBatchChangeSummaries.from_dict(data)
 
+    def approve_batch_change(self, batch_change_id, approval=None, **kwargs):
+        """
+        Approve a batch change
+
+        :return: the content of the response
+        """
+        url = urljoin(self.index_url, '/zones/batchrecordchanges/{0}/approve'.format(batch_change_id),
+                      to_json_string(approval))
+        response, data = self.__make_request(url, u'POST', self.headers, to_json_string(approval), **kwargs)
+
+        return BatchChange.from_dict(data)
+
+    def reject_batch_change(self, batch_change_id, rejection=None, **kwargs):
```

Comment:
running this against a local instance (master), im not seeing my review_comment or reviewer_username reflected in the GET result after rejecting

did:
```
>>> test = client.reject_batch_change("d720c35d-3201-4fb4-a0eb-9a8e649c50b8", "commented here!")
>>> test = client.get_batch_change("d720c35d-3201-4fb4-a0eb-9a8e649c50b8")
>>> test.approval_status
'ManuallyRejected'
>>> test.review_timestamp
datetime.datetime(2019, 8, 2, 17, 58, 12, tzinfo=tzutc())
>>> test.reviewer_username
>>> test.reviewer_id
'support-user-id'
>>> test.review_comment
>>> test.status
'Rejected'
```

Reply:
it's not a string it's a `RejectBatchChangeRequest` object and accept is an `AcceptBatchChangeRequest` object.

Reply:
updated this to take a string instead of an object